docs/iris/src/whatsnew/latest.rst

```diff
@@ -13,193 +13,255 @@ This document explains the changes made to Iris for this release
-  YAML files. See :pull:`3812`.
 
 Bugs Fixed
 ==========
 
-* The method :meth:`~iris.Cube.cube.remove_coord` would fail to remove derived
-  coordinates, will now remove derived coordinates by removing aux_factories.
+* `stephenworsley`_ fixed :meth:`~iris.Cube.cube.remove_coord` to now also
+  remove derived coordinates, by removing aux_factories. (:pull:`3641`)
 
-* The ``__iter__()`` method in :class:`~iris.cube.Cube` was set to ``None``.
-  ``TypeError`` is still raised if a :class:`~iris.cube.Cube` is iterated over
-  but ``isinstance(cube, collections.Iterable)`` now behaves as expected.
+* `Jon Seddon`_ fixed ``isinstance(cube, collections.Iterable)`` to now behave
+  as expected if a :class:`~iris.cube.Cube` is iterated over, while also
+  ensuring that ``TypeError`` is still raised. (Fixed by setting the
+  ``__iter__()`` method in :class:`~iris.cube.Cube` to ``None``).
+  (:pull:`3656`)
 
-* Concatenating cubes along an axis shared by cell measures would cause
-  concatenation to inappropriately fail.  These cell measures are now
-  concatenated together in the resulting cube.
+* `stephenworsley`_ enabled cube concatenation along an axis shared by cell
+  measures; these cell measures are now concatenated together in the resulting
+  cube. Such a scenario would previously cause concatenation to inappropriately
+  fail. (:pull:`3566`)
 
-* Copying a cube would previously ignore any attached
-  :class:`~iris.coords.CellMeasure`.  These are now copied over.
+* `stephenworsley`_ newly included :class:`~iris.coords.CellMeasure`s in
+  :class:`~iris.cube.Cube` copy operations. Previously copying a
+  :class:`~iris.cube.Cube` would ignore any attached
+  :class:`~iris.coords.CellMeasure`. (:pull:`3546`)
 
-* A :class:`~iris.coords.CellMeasure` requires a string ``measure`` attribute
-  to be defined, which can only have a value of ``area`` or ``volume``.
+* `Bill Little`_ set a :class:`~iris.coords.CellMeasure`'s
+  ``measure`` attribute to have a default value of ``area``.
   Previously, the ``measure`` was provided as a keyword argument to
-  :class:`~iris.coords.CellMeasure` with an default value of ``None``, which
-  caused a ``TypeError`` when no ``measure`` was provided.  The default value
-  of ``area`` is now used.
+  :class:`~iris.coords.CellMeasure` with a default value of ``None``, which
+  caused a ``TypeError`` when no ``measure`` was provided, since ``area`` or
+  ``volume`` are the only accepted values. (:pull:`3533`)
 
-* **All** plot types in `iris.plot` now use `matplotlib.dates.date2num
+* `Martin Yeo`_ set **all** plot types in `iris.plot` to now use
+  `matplotlib.dates.date2num
   <https://matplotlib.org/api/dates_api.html#matplotlib.dates.date2num>`_
   to format date/time coordinates for use on a plot axis (previously
   :meth:`~iris.plot.pcolor` and :meth:`~iris.plot.pcolormesh` did not include
-  this behaviour).
+  this behaviour). (:pull:`3762`)
```

Comment:
I haven’t checked, but it looks like this fixed #3347.

Reply:
I've now checked with `master` and #3347 is indeed fixed :sunglasses:.

Reply:
Wow, the "What's New" is working even before it gets merged 😂